src/vs/workbench/services/environment/browser/environmentService.ts

```diff
@@ -332,6 +333,26 @@ export class BrowserWorkbenchEnvironmentService implements IBrowserWorkbenchEnvi
 
 		return undefined;
 	}
+
+	@memoize
+	get filesToMerge(): IPath[] | undefined {
+		if (this.payload) {
+			const fileToMerge1 = this.payload.get('mergeFile1');
+			const fileToMerge2 = this.payload.get('mergeFile2');
+			const fileToMergeBase = this.payload.get('mergeFileBase');
+			const fileToMergeResult = this.payload.get('mergeFileResult');
+			if (fileToMerge1 && fileToMerge2 && fileToMergeBase && fileToMergeResult) {
+				return coalesce([
```

Comment:
There is nothing to coalesce, right?

Reply:
Yeah, I think that was from a time when I had `base` as optional but it no longer is. Maybe need a revisit but removed `coalesce` for now: https://github.com/microsoft/vscode/pull/155039/commits/abe9324b93abd7ad474e50015c4fb7e2c3af1d89